src/services/kratos/cron.ts

```diff
@@ -0,0 +1,34 @@
+import { kratosAdmin, listSessionsInternal } from "./private"
```

Comment:
this should be also in service with a normal structure...

Reply:
any example to follow?